nntrainer/meson.build

```diff
@@ -42,7 +42,8 @@ nntrainer_elements = [
   'optimizers',
   'tensor',
   'utils',
-  'graph'
+  'graph',
+  'opencl'
```

Comment:
This can be optional. how about adding ```option('enable-opencl', type: 'boolean', value: false)``` in meson_options.txt

Reply:
Added related changes in next commit. Updated description.